pyspark/graphar_pyspark/enums.py

```diff
@@ -0,0 +1,68 @@
+"""
+Copyright 2022-2023 Alibaba Group Holding Limited.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from enum import Enum
+
+from py4j.java_gateway import JavaObject
+
+from graphar_pyspark import GraphArSession
+
+
+class GarType(Enum):
+    """Main data type in gar enumeration."""
+    BOOL = "bool"
+    INT32 = "int32"
+    INT64 = "int64"
+    FLOAT = "float"
+    DOUBLE = "double"
+    STRING = "string"
+    LIST = "list"
+
+    @staticmethod
+    def from_scala(jvm_obj: JavaObject) -> "GarType":
```

Comment:
I changed it for other classes, but do you think that inheritance of Enum is meaningful? Because for enumerations I would have staticmethods better

Reply:
Thanks! I think it depends, in this particular case maybe you are right - people are likely to change this particular `Enum` instead of inheriting from it.